docs/cardinal/game/configuration/cardinal.mdx

```diff
@@ -18,7 +18,7 @@ TELEMETRY_TRACE_ENABLED = false
 
 ### BASE_SHARD_ROUTER_KEY
 
-A secure authentication token used for routing game shards. It ensures proper routing within the Cardinal system. 
+A secure authentication token used for routing game shards. It ensures proper routing within the Cardinal system.
```

Comment:
It's used to authenticated Cardinal with the EVM Base Shard gRPC private endpoint